meshnets/utils/data_visualization.py

```diff
@@ -1,16 +1,96 @@
-"""Load an object and visualize it using pyvista."""
+"""Methods for visualizing mesh objects."""
 
-from absl import logging
+from typing import Union
+from typing import Tuple
+import warnings
+
+import numpy as np
 import pyvista as pv
 
 
-def plot_data(obj_path: str, verbose: bool = False) -> None:
-    "Plot a mesh object with pyvista."
+def plot_mesh(mesh_path: str,
+              clim: Union[Tuple[float, float], None] = None,
+              rot_z: int = 0,
+              off_screen: bool = False,
+              screenshot: Union[str, None] = None) -> None:
+    """Plot a mesh object with its active scalar array."""
+
+    mesh = pv.read(mesh_path)
+    mesh = mesh.rotate_z(rot_z)
+    mesh.plot(show_edges=True,
+              cmap='RdBu_r',
+              clim=clim,
+              off_screen=off_screen,
+              screenshot=screenshot)
+
+
+def plot_mesh_with_scalars(mesh_path: str,
+                           scalars: np.array,
+                           clim: Union[Tuple[float, float], None] = None,
+                           rot_z: int = 0,
+                           off_screen: bool = False,
+                           screenshot: Union[str, None] = None) -> None:
+    """Plot a mesh object with a given scalar array."""
+
+    mesh = pv.read(mesh_path)
+    mesh = mesh.rotate_z(rot_z)
+
+    if scalars is None:
+        warnings.warn(
+            "Argument 'scalars' is None. Displaying with active scalar array.")
+
+    mesh.plot(scalars=scalars,
+              show_edges=True,
+              cmap='RdBu_r',
+              clim=clim,
+              off_screen=off_screen,
+              screenshot=screenshot)
+
+
+def plot_mesh_comparison(mesh_path: str,
+                         scalars_1: np.array,
```

Comment:
if this is the groundtruth, then call it so. Scalar_1 is not a very good name :)
The same for prediction.